tests/test_config_models.py

```diff
@@ -288,7 +277,7 @@ def test_key_values(self, mock_cache):
         self.assertEqual(len(unique_left_keys), 2)
         self.assertEqual(set(unique_left_keys), set(['left_a', 'left_b']))
 
-    def test_key_string_values(self, mock_cache):
+    def test_key_string_values(self):
         """ Ensure str() vs unicode() doesn't cause duplicate cache entries """
```

Comment:
Seems like this is more like a TieredCache test, so you may not need any of this. But, in theory, doing the same calls clearing the RequestCache in between would be needed to test the Django cache tier.